### DISCUSSION TITLE
9.9星期一的丰满

### DISCUSSION THREAD

**Discussion by Playmaker-hugo** [share121/inter-knot#4754] [灌水]
![20240910_200436](https://github.com/user-attachments/assets/92e7caf8-3b32-45c8-99a0-2e3c6b3fa498)
上一个帖子图片没显示出来，帖子也删不掉，重发一下。

**Comment by Emotional-Transparent** [comment]
![Screenshot_2024-09-10-22-33-45-20_219376f1dbde3085d7f592166ad0f554.jpg](https://github.com/user-attachments/assets/67c30643-fa5a-46ee-81c1-8fdedf31ab95)

这不能显示吗
